### DISCUSSION TITLE
Errore "Failed to start zigbee" - Configurazione Zigbee2mqtt

### DISCUSSION THREAD

**Discussion by Keishusan** [indomus/forum#1255] [HUB: Home Assistant]
Ciao a tutti,
premesso che ho un mini pc con Proxmox su cui ho installato Home Assistant, inoltre ho inserito una chiavetta SonOff Zigbee 3.0.

Seguendo la guida passo-passo, dopo aver fatto tutti i passaggi sono bloccato a questo punto con questo errore:
![1](https://user-images.githubusercontent.com/96801074/147890728-f032b622-5028-4097-9482-87ec92a126c1.JPG)

Questa è la mia configurazione:
![61d115e450a1b6 633185541641092580OeTRJigbtG5xsXzgG1x6](https://user-images.githubusercontent.com/96801074/147890712-511c97b8-be8a-4fcb-87da-1c372fa3d191.jpg)

La porta mi sembra corretta:
![Cattura](https://user-images.githubusercontent.com/96801074/147890764-12b45049-bb18-4353-8cf4-081a244b4dff.JPG)

Grazie mille

**Comment by snakuzzo** [comment]
Premesso che non conosco Proxmox, non dovrebbe cambiare nulla ma...prova a passargli `/dev/ttyUSB0`.
Ad ogni modo c'è una issue aperta [QUI](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/203).
Prova a scrivere lì o a seguire le indicazioni di chi ha risolto (mi pare di capire che pure lì si tratti della chiavetta di ITEAD)

**Reply by Keishusan** [reply]
Purtroppo ho provato anche quella porta e mi dà lo stesso errore, non può essere dovuto a qualche altro passaggio che posso aver sbagliato?
Ho guardato il link che mi hai scritto, ma mi sembra poi l'utente alla fine abbia degli errori diversi dal mio

**Reply by Keishusan** [reply]
Inoltre nel registro mi restituisce questo errore, non credo sia normale...
![222](https://user-images.githubusercontent.com/96801074/147891669-24e097fb-0690-455b-9d06-37b4f9ad7068.JPG)

**Reply by KC-inDomus** [reply]
usa `/dev/ttyUSB0`
ciò detto, fai una prova utilizzando zigbee2mqtt 1.18.x?

**Comment by gio-dot** [comment]
prova a riflashare la chiavetta

**Reply by Keishusan** [reply]
> 

la chiavetta è nuova, mai usata. Si fa anche in questi casi?

E, domanda più importante, come si fa?

**Reply by Keishusan** [reply]
> prova a riflashare la chiavetta

Ho trovato, ho fleshato, ma il problema rimane...
